### ISSUE TITLE
Amazon EU checkout issues

### ISSUE THREAD

**Issue by rutgermudde** [stellaraiop/StellarAIOV2#11]
Are there problems at the moment with the EU amazon site. Constantly getting softban / rotating proxy error (but account ist softbanned). Yesterday went back to 0.87, the problem was solved. Since a few hours I'm getting the errors again (but my accounts arent softbanned) Any suggestions Trying other (new) proxies don;t work (ISP Space proxies) Amazon accounts aren't softbanned But error is back up'. Also tried to login and out in session. All tasks are running in turbo mode (max 5 tasks per account at 4500 delay)

**Comment by Rockafella808** [comment]
Yeah same as rutgermudde it aint working properly for UK/EU, get softbanned on every Amazon account, then put those same accounts into another bot and they all checkout test purchases. Please fix it @Stella Devs...

**Comment by rutgermudde** [comment]
> Verstuurd vanaf mijn iPhone
> […](#)

Hey Rockafella, can we get in touch. Please mail me at rutgermudde@home.nl or discord user: rutgermudde#6291

**Comment by FrankieFrankske** [comment]
Same problem with Stellar for a few days. The new monitoring has never worked on the European and UK Amazons.

Now there are problems checking out with Turbo and with OfferID.

Turbo check out: Proxyban
OfferID ATC: Account ban
OfferID Fast: Proxyban
OfferID Frontend: OOS (Product is in stock)

Proxies and accounts are not banned after these notifications.

That way we can no longer use Stellar. Please fix it.

Thanks.

**Comment by selkk** [comment]
Amazon JP has the same problem as the above feedback appears, please fix it.
